### ISSUE TITLE
When run a job template, put the credential prompt in last place.

### ISSUE THREAD

**Issue by maydayuiui** [ansible/awx#8513]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY

We use Prompted crendential and Prompted Survey both. But our login method is to use RSA SecureID token which will change in a minute. In currently setting, survey always comes out after crendetial input. If the survey is long, we can't get the ssh connection success. Is it possible to put promted credential in last place? Thanks.

**Comment by AlexSCorey** [comment]
@maydayuiui it is possible to do that.  However, in new UI, the Prompt on Launch feature allows the user to navigate to any step in the launch process at any time.  So, you could, feasibly, open the prompt, skip the Credential Step, complete the survey step and then navigate back to the credential step.

**Comment by maydayuiui** [comment]
@AlexSCorey Does Version 15.0.0 use the new UI? I'm using 15.0.0, but I can't navigate to Survey if i don't fill in credential first. I only can navigate back to the credential step after finish all the prompt steps.

**Comment by AlexSCorey** [comment]
@maydayuiui Yes see the [README](https://github.com/ansible/awx/blob/devel/awx/ui_next/README.md#Development ) for UI_Next
